aten/src/ATen/native/cudnn/Conv.cpp

```diff
@@ -789,6 +796,16 @@ static inline void split_batch_dim_to_32bit_out(
 }
 
 
+#if defined(CUDA_VERSION) && CUDA_VERSION >= 11000
```

Comment:
Question where I'm not sure of the right answer:  should we assert on `CUDA_VERSION` or `CUDNN_VERSION` here?

In general, is it safe to assume we're rolling forward Cuda and Cudnn together, such that `CUDA_VERSION >= 11` guarantees `CUDNN_VERSION >= 8000` and vice versa (put another way, we don't ever plan to build with cuda 10 and cudnn 8, or cuda 11 and cudnn 7.x)?

After discussions with @ptrblck the answer seems to be yes, we do want to guarantee that cudnn and cuda roll forward together, but I'm not sure.

(If the answer is yes, it doesn't matter whether we assert on `CUDA_VERSION` or `CUDNN_VERSION`.)

Reply:
We should use `CUDNN_VERSION` consistently everywhere.